image_topic: "/rscamera_front/color/image"
cmd_vel_topic: "/cmd_vel"
server_url: "http://127.0.0.1:8000"
linear_speed: 0.25
angular_speed: 0.5
ramp_duration: 0.2
ramp_steps: 5
request_timeout: 5.0
default_instruction: "Please navigate according to the current mission."
